config/config.go

```diff
@@ -128,6 +130,8 @@ const (
 	WHITELISTED_HEADERS_ENVIRONMENT_KEY                               = "WHITELISTED_HEADERS"
 	DEFAULT_ACCESS_CONTROL_ALLOW_ORIGIN_VALUE_ENVIRONMENT_KEY         = "DEFAULT_ACCESS_CONTROL_ALLOW_ORIGIN_VALUE"
 	HOSTNAME_TO_ACCESS_CONTROL_ALLOW_ORIGIN_VALUE_MAP_ENVIRONMENT_KEY = "HOSTNAME_TO_ACCESS_CONTROL_ALLOW_ORIGIN_VALUE_MAP"
+	PPROF_USERNAME_ENVIRONMENT_KEY                                    = "PPROF_USERNAME"
```

Comment:
since these will be removed eventually, thoughts on using the db user/db pass instead since db password is already in ssm

Reply:
@pirtleshell thoughts?

Reply:
eh, just gonna add it